kernel/src/main/scala/org/apache/toree/boot/layer/ComponentInitialization.scala

```diff
@@ -122,8 +123,12 @@ trait StandardComponentInitialization extends ComponentInitialization {
   }
 
   private def initializeDependencyDownloader(config: Config) = {
-    val dependencyDownloader = new IvyDependencyDownloader(
+    /*val dependencyDownloader = new IvyDependencyDownloader(
       "http://repo1.maven.org/maven2/", config.getString("ivy_local")
+    )*/
+    val dependencyDownloader = new CoursierDependencyDownloader
+    dependencyDownloader.setDownloadDirectory(
+      new File(config.getString("ivy_local"))
```

Comment:
If we change this to something else, we could avoid clashes between kernel instances. Or, if we set the default ivy_local to a temporary directory, that would work as well.